toolchain/lc-compile/src/emit.cpp

```diff
@@ -425,7 +425,44 @@ void EmitForeignHandlerDefinition(long p_index, PositionRef p_position, NameRef
     else
         t_binding_str = to_mcstringref(p_binding);
     
-    MCScriptAddForeignHandlerToModule(s_builder, to_mcnameref(p_name), p_type_index, *t_binding_str, p_index);
+    if (strncmp((const char *)p_binding, "lcb:", 4) != 0)
```

Comment:
Hah - this is case of me forgetting the emit code in the compiler is actually C++ and not C